src/services/ui/src/components/Layout/index.tsx

```diff
@@ -114,7 +114,14 @@ const ResponsiveNav = ({ isDesktop }: ResponsiveNavProps) => {
       <>
         {getLinks(!!data.user).map((link) => (
           <NavLink
-            to={link.link}
+            to={
+              link.name !== "FAQ" && link.name !== "Dashboard" ? link.link : ""
+            }
+            onClick={() => {
+              if (link.name === "FAQ" || link.name === "Dashboard") {
+                window.open(link.link, "myWindow");
```

Comment:
The logic in `to` and `onClick` get pretty complex. I think this can be solved with the same solution from below where you set `target` to a custom value. 

I also experienced two bugs: 

1. All NavLinks come styled with underlines as if they're being hovered or are active. Clearing up this logic with the simpler approach may help with that, too.

<img width="172" alt="Screen Shot 2023-10-04 at 2 51 08 PM" src="https://github.com/Enterprise-CMCS/macpro-mako/assets/4022304/78b50ceb-aa4c-42d2-9b3e-bf0b72619f32">

2. Links using this onClick approach don't open in the same tab as the link using the `target` approach found below.